core/src/apps/cardano/certificates.py

```diff
@@ -7,7 +7,7 @@
     get_public_key_hash,
     validate_reward_address,
 )
-from .helpers import INVALID_CERTIFICATE, LOVELACE_MAX_SUPPLY
+from .helpers import INVALID_CERTIFICATE, LOVELACE_MAX_SUPPLY, PUBLIC_KEY_HASH_SIZE
```

Comment:
I've not checked across all the codebase, but having a basic definition such as `PUBLIC_KEY_HASH_SIZE` in `helpers` seems fishy. I would expect something more like `cardano.h` for ledger which groups all those external constants. Also, the name used there is `ADDRESS_KEY_HASH_LENGTH` --- there are other public keys around, e.g. VRF key is hashed to 32 bytes, so we want something more specific.

Reply:
Agree, renamed in https://github.com/vacuumlabs/trezor-firmware/pull/60/commits/b5c1ee5fe843ec8daed762b157e6e8d7db49b168. I have kept it in the helpers module though. It doesn't seem to be a convetion to have constants in the `__init__.py` of the app (cardano) module (which is IMHO equivalent to `cardano.h`). If there were more constants used throughout the Cardano module we could introduce a `consts.py` file or something similar, but I don't think that's necessary for the two values we currently have.